modules/bridge/src/main/ts/ephox/bridge/components/menu/Image.ts

```diff
@@ -0,0 +1,28 @@
+import { FieldSchema, ValueSchema } from '@ephox/boulder';
+import { Optional, Result } from '@ephox/katamari';
+
+export interface ImageSpec {
+  type: 'image';
+  src: string;
+  classes?: string[];
```

Comment:
Likewise, classes is normally an attribute we don't want to expose here.

Reply:
See https://github.com/tinymce/tinymce/pull/6149#discussion_r515868104